src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -145,13 +147,13 @@ private IEnumerable<DataColumn> columnsFor(CurveData<TimeProfileXValue, TimeProf
          if (curveData.IsRange())
             return new[] 
             {
-               new DataColumn(" (Lower)", curveData.YAxis.Dimension, baseGrid)
```

Comment:
I don't understnad the code on the right or on the left.  " (Lower)" ??!? Was it like this before?

Reply:
It was already merged in some previous PR. This is for the ranges. So, before we had only one value for a range parameter. In case we have 5-95 percentile it will only show the lower value (5 percentile). Now, it adds the two values as two separate columns. The first column will be <5-95 percentile> (Lower) and the second will be <5-95 percentile> (Upper). I realized it was coded using the string built on the fly and moved it to the constants (and reuse it in the table headers for both calculations - on individuals and on curves).